### ISSUE TITLE
Please mention that we need to get rid off every software that was in contact with the FSF

### ISSUE THREAD

**Issue by Sven792** [rms-open-letter/rms-open-letter.github.io#1904]
It is important to mention that we need to cancel every software project that had something to do with the FSF and every other organization that have political ideas (like Mozilla, GNOME Foundation, Tor Project and etc.) because it is a disaster if they have opinions. We need to restrict free speech and punish people if they express their opinions! /s

**Comment by ppbdrinker** [comment]
Agreed. For the greater justice it is important to eradicate anyone possessing at least a tiny bit of brains.
